### DISCUSSION TITLE
VScode有Jupyter擴充套件！

### DISCUSSION THREAD

**Discussion by Yunhans** [1101-GS4719-C/Python-Community-GS4719-C#689] [General]
VScode真的太扯了，連Jupyter notebook都有
只需要把檔名改成.ipynb就行
<img width="1440" alt="截圖 2022-09-21 下午3 10 11" src="https://user-images.githubusercontent.com/91072133/191438200-aacdb16d-cd3f-4e78-92b2-449cf48ddc87.png">
<img width="1440" alt="截圖 2022-09-21 下午3 31 09" src="https://user-images.githubusercontent.com/91072133/191442436-319bf172-0cd6-403d-8da3-c60282b4659c.png">

**Comment by danwan1027** [comment]
太扯了 以後不用再那麼麻煩的打開終端機了！！

**Comment by Chiway1229** [comment]
我覺得這種言簡意賅的風格有點東西
不過身為VScode使用者
感謝你分享這種實用的程式內插件

**Comment by bonny123865** [comment]
讚喔讚喔很方便!!
可愛的小知識又增加了呢

**Comment by a45320614** [comment]
太好了! 我剛好需要在VScode上使用Jupyter Notebook!

**Comment by wangtyphoon** [comment]
vs code 的互動式視窗我覺得也蠻不錯的

**Comment by Jasper525** [comment]
Anaconda也有提供jupyter可以使用
他的jupyter不需要額外下載，在anaconda主頁點launch jupyter notebook會在瀏覽器新增一個jupyter分頁
供喜歡jupyter的人參考

**Comment by williamyen0623** [comment]
vscode的功能強大，連這個都可以做到！

**Comment by manipulateed** [comment]
vscode 真的是個很多元便捷的原始碼編輯器，附有非常多的語言和軟體，以一抵百阿。

**Comment by WEI-CHEN-CHUN** [comment]
VS Code 作為最強大的編輯器當之無愧
其快捷鍵非常方便

**Comment by Zoe06060606** [comment]
VS Code真的很方便
支援超多程式語言 要嵌入各種套件也很容易

**Comment by ruthchen1020** [comment]
要卸載anaconda 了👍

**Comment by AnAnistired** [comment]
原本就有在用vs學c（初學），但不知道原來還有這麼多功能🥰

**Comment by evanchou1204** [comment]
這樣好讚，不然每次改code執行檔又卡住真的會瘋掉。

**Comment by Pietra1226** [comment]
我用VScode寫了C++/python/Java/HTML，介面真的很簡單舒服

**Comment by Rodrigo-dev-bit** [comment]
謝謝分享，但是我還是習慣直接開jupyter，覺得比較方便

**Comment by dabbie109303501** [comment]
原來vscode還有這種功能，難怪是工程師很喜歡用的軟體

**Comment by zhiron544** [comment]
我覺得VScode很強大，支援Jupyter Notebook很棒，希望能和其他Python環境相容。
